package.json

```diff
@@ -56,5 +56,5 @@
   "lint-staged": {
     "*.js": "lint"
   },
-  "proxy": "http://localhost:8000"
+  "proxy": "http://54.93.203.184"
```

Comment:
I believe the proxy should be used for DEV purposes.
[Documentation](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#proxying-api-requests-in-development)

Essentially the `proxy` property is expecting a setup where the frontend and backend live in the same domain, or at least API requests are directed to the same domain.

So if I'm not mistaken, this line should be removed from `package.json` and some endpoint logic should be added in the source files.
Preferably some logic where I could run something like `yarn run start:local` if I want it to run against my local instance of the backend

Reply:
Ill look into this